mne/utils/docs.py

```diff
@@ -384,7 +384,7 @@
     String format for colorbar values.
 """
 docdict["topomap_mask"] = """
-mask : ndarray of bool, shape (n_channels, n_times) | None
+mask : ndarray of bool, shape (n_channels, n_samples) | None
     The channels to be marked as significant at a given time point.
     Indices set to ``True`` will be considered. Defaults to ``None``.
```

Comment:
I think here we should be more explicit than to say "channels marked as significant".
```suggestion
mask : ndarray of bool, shape (n_channels, n_samples) | None
    Array indicating channel-time combinations to highlight with a distinct
    plotting style. Indices set to ``True`` will be plotted with the parameters
    given in ``mask_params``. Defaults to ``None``.
```

Reply:
Sure, but I'm not sure this docstring is good either. `mask` highlights sensors on a given topography at a given time. In your proposition, it is not clear what is highlighted. 

Also, instead of channels, I would use the word sensor to link to the `sensors` argument used for plotting. 

Maybe something along the lines of:

> Sensor-time combinations array indicating the sensors to highlight with a distinct plotting style at a given time point.

Reply:
> Also, instead of channels, I would use the word sensor to link to the `sensors` argument used for plotting.

The "sensors" that are plotted are not necessarily all of the channels in the Evoked instance (since user can pass a `ch_type` argument --- but (I think) even if they do restrict the `ch_type` the mask still has to be the same dimension as `evoked.data` (in other words, the mask must account for all *channels* not just the subset of *sensors* that are drawn on the plot).  That's why the docstring of this parameter uses "n_channels" in the shape specification. 

> Maybe something along the lines of:
> 
> > Sensor-time combinations array indicating the sensors to highlight with a distinct plotting style at a given time point.

To me that is not more clear than my past proposal.  What about this:

```
mask : ndarray of bool, shape (n_channels, n_samples) | None
    Array indicating channel-time combinations to highlight with a distinct
    plotting style (useful for, e.g., marking which channels at which times a
    statistical test of the data reaches significance). Array elements set to
    ``True`` will be plotted with the parameters given in ``mask_params``.
    Defaults to ``None``, equivalent to an array of all ``False`` elements.
```

Reply:
Let's keep in mind that this is not only for evoked instances; but for all topographic map plots. This now leads me to question this `ch_type` argument. 

> [...] not necessarily all of the channels in the Evoked instance (since user can pass a `ch_type `argument [...]

So for an Evoked instance with multiple channels types, the user can pick channels to plot with `ch_type`?
However, [`mne.viz.plot_topomap()`](https://mne.tools/dev/generated/mne.viz.plot_topomap.html#mne.viz.plot_topomap), which is the function called in the end by `evoked.plot_topomap()`, looks different with the argument `pos` which must be an `Info` **with only one channel type**. Yet, it still has a `ch_type` argument with the same description. I'm getting a bit confused as to what `ch_type` does.

Back to the matter at hand, I will check tomorrow that the dimension of the mask must include all the channels present in the Evoked object and not only the channels plotted. 
The last proposition does look better to me.

Reply:
> So for an Evoked instance with multiple channels types, the user can pick channels to plot with `ch_type`?

yes

> However, [`mne.viz.plot_topomap()`](https://mne.tools/dev/generated/mne.viz.plot_topomap.html#mne.viz.plot_topomap), which is the function called in the end by `evoked.plot_topomap()`, looks different with the argument `pos` which must be an `Info` **with only one channel type**. Yet, it still has a `ch_type` argument with the same description.

It should not have the same description. The docstring of `plot_evoked_topomap` should not say "The channel type being plotted. Determines the 'auto' extrapolation mode."  The problem appears to be that the `topomap_ch_type` key appears twice in the docdict.  Can you update the key [here](https://github.com/mne-tools/mne-python/blame/main/mne/utils/docs.py#L321) to be `evoked_topomap_ch_type` and also update the key used in the `plot_evoked_topomap` docstring?
 
> Back to the matter at hand, I will check tomorrow that the dimension of the mask must include all the channels present in the Evoked object and not only the channels plotted.

Yes please do double-check that, thanks

Reply:
Ok, that makes more sense. I will update the docdict key tomorrow.

Reply:
Also, now that I am looking at this `ch_type` argument.. 
In 0.23, for the function [`mne.viz.plot_topomap()`](https://mne.tools/stable/generated/mne.viz.plot_topomap.html?highlight=mne%20viz%20plot_topomap#mne.viz.plot_topomap), the `New in version` says `0.21`.

    The channel type being plotted. Determines the 'auto' extrapolation mode.
    New in version 0.21.

But now, in the code of 0.24, the doc is created with:

```
    %(topomap_ch_type)s

        .. versionadded:: 0.24.0
```

This leads to the very weird:

![image](https://user-images.githubusercontent.com/73893616/128424379-dd01d73b-f427-4db2-b960-1c9fa6251e91.png)

Any thoughts on this? Let's fix it at the same time.
`plot_topomap()` does appear to have a new argument, `cnorm` in 0.24, which is correctly annotated with `.. versionadded:: 0.24`. Typo which leads to a similar addition for an already existing argument?

Reply:
Yep, typo from the addition of `cnorm` I think: https://github.com/mne-tools/mne-python/commit/b6a36089a963a97380488845491fe38e8d3b610b

Reply:
nice detective work @mscheltienne

Reply:
I've updated the docstring suggestion to use my later wording.  with that and the removal of the spurious `versionadded` line I think this will be good to go.